libs/designer/src/lib/core/state/operation/operationMetadataSlice.ts

```diff
@@ -119,6 +124,9 @@ const initialState: OperationMetadataState = {
   staticResults: {},
   repetitionInfos: {},
   errors: {},
+  loadStatus: {
+    nodesInitialized: false,
```

Comment:
A little orthogonal to this PR itself but can we add some logging to time how long after load starts to this switching to true? I think that would be useful data to have.